hooks/lib/pdlc-session.sh

```
@@ -0,0 +1,161 @@
+# Usage: pdlc_session_save <spec_dir> <iteration> <lifecycle_state> \
+#            <director_decision> <actor_result> <critic_verdict>
+#
+# Arguments:
+#   spec_dir          — path to the spec directory
+#   iteration         — current iteration number (integer)
+#   lifecycle_state   — inferred lifecycle state (e.g., Implementing)
+#   director_decision — "action|mode|rationale" string
+#   actor_result      — summary of Actor outcome
+#   critic_verdict    — accept, retry, or escalate
+pdlc_session_save() {
+  local spec_dir="$1"
```

Shellcheck (0.11.0)

[warning] spec_dir appears unused. Verify use (or export if used externally).

(SC2034)